README.md

```diff
@@ -9,10 +9,50 @@ Check the state of a openSUSE MicroOS system after a reboot.
 process.
 
 The `health-checker` script will call several plugins. Every plugin is
-responsible to check a special service or condition. All services, which
-should be checked by the plugin, needs to be listed in the 'After' section.
-To run the check the plugin is called with the option *check*. If this fails,
-the plugin will exit with the return value `1`, else `0`.
+responsible to check a special service or condition. All services, which should
+be checked by the plugin, needs to be listed in the 'After' section. To run the
+check the plugin is called with the option *check*. If this fails, the plugin
+will exit with the return value `1`, else `0`. The `health-checker` script will
+call all plugins inside `/usr/libexec/health-checker` and
+`/usr/local/libexec/health-checker` directories. The former directory includes plugins
+installed by system packages (and therefore coming through an RPM), the latter includes
+plugins installed manually by the system admin. Every plugin is responsible to check
+a special service or condition. For this, the plugin is called with the option
+*check*. If this fails, the plugin will exit with the return value `1`, else
+`0`. Have a look at the default plugins shipped in
+`/usr/libexec/health-checker` for examples.
+
+Its behavior depends if the system is using systemd-boot/grub2-bls (i.e.
```

Comment:
Better: "is using systemd-boot, grub2-bls or any other bootloader following the Boot Loader Specification (BLS) or legacy..."
I wouldn't mention bootloader internal things like the /boot/efi/loader/entries, this can change in the future and a change is already under discussion.